Server/EVESharp.Node/Services/Inventory/ship.cs

```diff
@@ -14,161 +16,212 @@
-    private         IDogmaNotifications DogmaNotifications { get; }
-    private         IDatabase           Database           { get; }
-    private         IDogmaItems         DogmaItems         { get; }
-
-    public ship
-    (
-        IItems        items, IBoundServiceManager manager, ISessionManager sessionManager, IDogmaNotifications dogmaNotifications, IDatabase database,
-        ISolarSystems solarSystems, IDogmaItems dogmaItems
-    ) : base (manager)
+    public override AccessLevel AccessLevel => AccessLevel.None;
+    private ItemEntity Location { get; }
+    private IItems Items { get; }
+    private ITypes Types => this.Items.Types;
+    private ISolarSystems SolarSystems { get; }
+    private ISessionManager SessionManager { get; }
+    private IDogmaNotifications DogmaNotifications { get; }
+    private IDatabase Database { get; }
+    private IDogmaItems DogmaItems { get; }
+
+    public ship(
+        IItems items, IBoundServiceManager manager, ISessionManager sessionManager, IDogmaNotifications dogmaNotifications,
+        IDatabase database, ISolarSystems solarSystems, IDogmaItems dogmaItems
+    ) : base(manager)
     {
-        Items              = items;
-        SessionManager     = sessionManager;
+        Console.WriteLine("[DEBUG] Node.Services.Inventory.ship (global service) constructed");
+        Items = items;
+        SessionManager = sessionManager;
         DogmaNotifications = dogmaNotifications;
-        Database           = database;
-        SolarSystems       = solarSystems;
-        DogmaItems         = dogmaItems;
+        Database = database;
+        SolarSystems = solarSystems;
+        DogmaItems = dogmaItems;
     }
 
-    protected ship
-    (
-        ItemEntity location, IItems items, IBoundServiceManager manager, ISessionManager sessionManager, IDogmaNotifications dogmaNotifications, Session session,
-        ISolarSystems solarSystems, IDogmaItems dogmaItems
-    ) : base (manager, session, location.ID)
+    protected ship(
+        ItemEntity location, IItems items, IBoundServiceManager manager, ISessionManager sessionManager,
+        IDogmaNotifications dogmaNotifications, Session session, ISolarSystems solarSystems, IDogmaItems dogmaItems
+    ) : base(manager, session, location.ID)
     {
-        Location           = location;
-        Items              = items;
-        SessionManager     = sessionManager;
+        Console.WriteLine("[DEBUG] Node.Services.Inventory.ship (bound instance) constructed for objectID=" + location.ID);
+        Location = location;
+        Items = items;
+        SessionManager = sessionManager;
         DogmaNotifications = dogmaNotifications;
-        SolarSystems       = solarSystems;
-        DogmaItems         = dogmaItems;
+        SolarSystems = solarSystems;
+        DogmaItems = dogmaItems;
     }
 
-    public PyInteger LeaveShip (ServiceCall call)
+    public PyInteger LeaveShip(ServiceCall call)
```

Comment:
A lot of these changes seem to boil down to formatting and should be excluded unless there's actual logic changes.